### ISSUE TITLE
Data for Massachusetts displaying weird

### ISSUE THREAD

**Issue by stephen304** [aatishb/covidtrends#213]
Looking at the data, it seems normal (all around 120k), so maybe this is just a bug in the graph. It looks like the line is going to negative infinity (I can't find the end) @aatishb 

![image](https://user-images.githubusercontent.com/981915/92308158-a78f1c80-ef69-11ea-8f31-8221bb07ed20.png)

**Comment by drlaurenwasson** [comment]
Came here to leave the same comment. Thanks for beating me to it!

**Comment by DesiOtaku** [comment]
I tried to see what could be causing the bug but I can't seem to find it.

Only lead I have so far is that it is downloading the data directly from https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv and the Massachusetts data looks fine there.

**Comment by stephen304** [comment]
The data looks extra weird today, it looks like the cumulative confirmed cases must have decreased in the data, as Massachusetts moved left on the x axis:

![image](https://user-images.githubusercontent.com/981915/92752785-3e3b4f00-f357-11ea-9002-23b752c9e59b.png)
@aatishb

**Comment by aatishb** [comment]
Hi all, Thanks for filing and looking into this. It looks like Massachusettes changed the way their data is being reported which resulted in fewer case counts. It's a bit easier to see on the linear scale view.

https://github.com/nytimes/covid-19-data/issues/447
https://www.nytimes.com/interactive/2020/us/massachusetts-coronavirus-cases.html#anomaly-notes
https://www.mass.gov/doc/covid-19-dashboard-september-2-2020/download

Since it's a data source / methodology issue we don't really have a great way of dealing with this on our end. We can wait and see if NYT decides to retroactively update their data to account for the methodology.

**Comment by stephen304** [comment]
@aatishb Does that also explain the graph not showing any points for 9/2-9/8? There should be data for those days but no points exist on the graph

Edit: Just realized that I was looking at the wrong number in the data - I guess the negative change in the data is what caused the missing points

**Comment by aatishb** [comment]
@stephen304 Exactly. It seems the large negative change in numbers on 9/2 led to the 'change in the previous week' as being negative for the entire next week (9/2 - 9/8).

A potential workaround might be to drop negative values before graphing, that way the graph would instead show a jump from 9/1 to 9/9. However if this is the only place this is occurring it might be worth waiting to see if they update the data source first.

**Comment by vjandrea** [comment]
Hello, the graph looks bizarre for France:

![image](https://user-images.githubusercontent.com/1639757/123654033-f88c3c00-d82d-11eb-8e33-246bf479c7e4.png)


and Spain shows a drop:

![image](https://user-images.githubusercontent.com/1639757/123654234-28d3da80-d82e-11eb-8b7a-fc42beea538a.png)
